### ISSUE TITLE
use NSTextContainer from Notepad initializer

### ISSUE THREAD

**Issue by DivineDominion** [ruddfawcett/Notepad#18]
`Notepad.init(frame: CGRect, textContainer: NSTextContainer?)` does not actually use `textContainer`. This at least violates obvious expectations :) (And the Liskov Substitution Principle)

**Comment by ruddfawcett** [comment]
Thanks for filing issue. If my memory serves me correctly, I think that that was intentional... Perhaps it's a required initializer?

**Comment by DivineDominion** [comment]
Yes it is, but if you call `Notepad(frame: someFrame, textContainer: myTextContainer)`, then `myTextContainer` will be discarded and replaced by another one. While I don't like to `fatalError` out of situations like this for various reasons (same rationale as for [partial functions](https://www.cocoawithlove.com/blog/2016/01/25/partial-functions-part-one-avoidance.html)), that would at least be less surprising and easy to notice. 

Simply _using_ the passed-in text container would be best, of course :) I don't know, yet, if that's not possible at all. And this is not very pressing an issue for me personally, so I'm okay with letting this sit here for a while.

**Comment by ruddfawcett** [comment]
Totally makes sense regarding the discarding of the passed in `textContainer`. I'm fine with letting this sit for a little bit as well, as I think I tried a few things that didn't work in the past.
